### ISSUE TITLE
Convert tensorflow model err

### ISSUE THREAD

**Issue by ChisenZhang** [alibaba/MNN#63]
./MNNConvert -f TF --modelFile ./frozen_model.pb --MNNModel testTRmodel.mnn --bizCode MNN

MNNConverter Version: 0.0.0.1 - MNN @ 2018

Start to Convert Other Model Format To MNN Model...
terminate called after throwing an instance of 'Error'
  what():  [10:45:39] /data1/image_data/data/online_pushed_data/parse_result/illegalPicCls/MNN/tools/converter/source/tensorflow/FusedBatchNormTf.cpp:46: Check failed: channels > 0 ==> Batchnorm Channels Paramter is Wrong! 0 name: MobileNet/batch_normalization/FusedBatchNorm
Aborted (core dumped)

是我模型有问题吗？  但是tensorflow跑起来没有错误呀，pb文件也测试过

**Comment by czy2014hust** [comment]
Check the BN's input nodes(especially variance), variance should be a vector with length of channels, not one scalar.

**Comment by ChisenZhang** [comment]
> Check the BN's input nodes(especially variance), variance should be a vector with length of channels, not one scalar.

Thanks!

**Comment by rkshuai** [comment]
@ChisenZhang 请问你解决了吗？

**Comment by rkshuai** [comment]
@czy2014hust  怎么解决呢？

**Comment by ChisenZhang** [comment]
> @ChisenZhang 请问你解决了吗？

解决了，当时BN的参数存错了，不知道你什么问题

**Comment by diligent321** [comment]
hello, a FusedBatchNorm layer with only one channel is included in my network, as the following image indicated. By the way, it's a segmentation task, so when I run MNNConvert command,  a similar fault raised, what can i do  to solve the problem? @czy2014hust
![捕获](https://user-images.githubusercontent.com/31379237/61100371-83849800-a498-11e9-9114-079269dd95cf.PNG)

**Comment by mingqizhang** [comment]
请问Pb文件中带有Bn层怎么正确转化到mnn模型，转换之前需要先将bn层处理下吗?
